packages/paste-core/components/toast/src/Toast.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {safelySpreadBoxProps} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {CloseIcon} from '@twilio-paste/icons/esm/CloseIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+import {MediaObject, MediaFigure, MediaBody} from '@twilio-paste/media-object';
+import {ErrorToast} from './ErrorToast';
+import {NeutralToast} from './NeutralToast';
+import {SuccessToast} from './SuccessToast';
+import {WarningToast} from './WarningToast';
+import {ToastProps, ToastPropTypes, ToastVariants} from './types';
+import {ToastVariantObject} from './constants';
+
+const ToastComponentVariants = {
+  error: ErrorToast,
+  neutral: NeutralToast,
+  success: SuccessToast,
+  warning: WarningToast,
+};
+
+const renderToastIcon = (variant: ToastVariants): React.ReactElement => {
+  switch (variant) {
+    case ToastVariantObject.ERROR:
+      return <ErrorIcon color="colorTextError" decorative={false} title="error: " size="sizeIcon20" />;
+    case ToastVariantObject.SUCCESS:
+      return <SuccessIcon color="colorTextSuccess" decorative={false} title="success: " size="sizeIcon20" />;
+    case ToastVariantObject.WARNING:
+      return <WarningIcon color="colorTextWarning" decorative={false} title="warning: " size="sizeIcon20" />;
+    case ToastVariantObject.NEUTRAL:
+    default:
+      return <InformationIcon color="colorTextNeutral" decorative={false} title="information: " size="sizeIcon20" />;
+  }
+};
+
+const Toast: React.FC<ToastProps> = ({children, hidden, onDismiss, variant = 'neutral', ...props}) => {
+  const ToastComponent = ToastComponentVariants[variant];
+
+  return (
+    <ToastComponent hidden={hidden} role="status" variant={variant} {...safelySpreadBoxProps(props)}>
```

Comment:
@SiTaggart am I on the right track here by providing a prop to hide an individual toast? Then showing it via button click. See here: https://github.com/twilio-labs/paste/pull/760/files#diff-cec5e6eba87fe35785110f3a7946cd6cR148

Reply:
Removed the hidden prop since its a valid HTML attribute.